README.md

```diff
@@ -2,3 +2,16 @@ Test Repository
 ===============
 
 Playing around with Git and GitHub.
+
+This is my modification, Claudia.
+
+This week I am in Suse Nurnberg.
+More modifications into Readme.md
+
+--
+
+Cities
```

Comment:
Could you make a section of that? Use "-" underlines.

Reply:
Modified according to your suggestions.